frontend/pages/interest/thank-you.tsx

```diff
@@ -0,0 +1,24 @@
+import { Box, Stack, Typography } from '@mui/material';
+import Lottie from 'lottie-react';
+import { NextPage } from 'next';
+import orangeTick from '../../assets/orangeTickLottie.json';
+import { lottieSx, stackSx } from '../../styles/interest/thankyou';
+
+const ThankYou: NextPage = () => {
+  return (
+    <Stack component="main" maxWidth="md" spacing={2} sx={stackSx}>
+      <Box sx={lottieSx}>
+        <Lottie autoPlay loop={false} animationData={orangeTick} />
+      </Box>
+      <Typography variant="h1" textAlign="center">
+        Thank you for getting in touch!
+      </Typography>
+      <Typography variant="body1" textAlign="center">
+        We are excited to find new partners and bring the gift of giving to more people.
```

Comment:
Maybe we can make the thank you sound more personal, but it's up to you. KIV for future if we make this component within the interest form page in future, we can use their name too

```suggestion
        We are excited to work with you to bring the gift of giving to more people.
```